### PR TITLE
enable LUA debugging

### PR DESCRIPTION
**Pull request by andreas-volz**
Support for 
https://github.com/LuaDist/mobdebug

### CODE REVIEW

File: src/video/font.cpp [hunk 1]
```diff
@@ -256,8 +256,8 @@ static bool GetUTF8(const std::string &text, size_t &pos, int &utf8)
 		utf8 |= (c & 0x3F);
 	}
 
-	int ascii = utf8_to_ascii(utf8);
-	utf8 = ascii;
+	//int ascii = utf8_to_ascii(utf8);
+	//utf8 = ascii;
```
Comment (timfel) [inline src/video/font.cpp:260]:
```suggestion
        int ascii = utf8_to_ascii(utf8)
	utf8 = (ascii >= 32 && ascii < 256) ? ascii : utf8;
```

File: src/video/font.cpp [hunk 2]
```diff
@@ -311,8 +311,8 @@ static bool GetUTF8(const char text[], const size_t len, size_t &pos, int &utf8)
 		utf8 |= (c & 0x3F);
 	}
 	
-	int ascii = utf8_to_ascii(utf8);
-	utf8 = ascii;
+	//int ascii = utf8_to_ascii(utf8);
```
Comment (timfel) [inline src/video/font.cpp:314]:
```suggestion
        int ascii = utf8_to_ascii(utf8)
	utf8 = (ascii >= 32 && ascii < 256) ? ascii : utf8;
```

File: src/video/font.cpp [hunk 3]
```diff
@@ -843,7 +843,8 @@ void CFont::MeasureWidths()
 		while (sp < gp) {
 			// Some accented glyphes are not perfectly aligned on the glyph grid (like ï or î ).
 			// So we need to do -1 to not compute width on the next glyph.
-			const unsigned char *lp = sp + (G->Width - 1); 
+			//const unsigned char *lp = sp + (G->Width - 1);
```
Comment (timfel) [inline src/video/font.cpp:846]: This is actually needed for french and german to work on my machine, to you have problems with this?

Comment (andreas-volz) [reply src/video/font.cpp:846]: I reverted the complete commit and now german (including display of umlauts) works and no crash. I didn't check each single change in this commit. Should I enable some single lines to test it? (Hint: I didn't test if 'ß' works. Need to find an intro with this letter.

Comment (andreas-volz) [reply src/video/font.cpp:846]: Ah sorry and in the pullrequest are both, LUA and the german font issue. My bad! Should I reject and change this pullrequest?

Comment (andreas-volz) [reply src/video/font.cpp:846]: I could confirm 'ß' not working. I'll close and open a new one

**Pull request closed without merging**